rdflib/namespace/__init__.py

```diff
@@ -590,6 +590,34 @@ def compute_qname_strict(
 
             return self.__cache_strict[uri]
 
+    def expand_qname(self, curie: str) -> Union[str, None]:
```

Comment:
I would rather have this expand_curie, as curie is a superset of QName:

https://www.w3.org/TR/curie/

> This specification addresses the problem by creating a new data type whose purpose is specifically to allow for the definition of scoped names that map to URIs in exactly this way. This type is called a "CURIE" or a "Compact URI". Syntactically, CURIEs are a superset of QNames.

Reply:
Ah so. np, changed.